Sources/Private/Model/DictionaryInitializable.swift

```diff
@@ -8,7 +8,7 @@
 // MARK: - InitializableError
 
 enum InitializableError: Error {
-  case invalidInput
+  case invalidInput(file: StaticString = #file, line: UInt = #line)
```

Comment:
Adding in file and line context makes it easier to debug where these errors are coming from